package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
This is not correct – `jsnext:main` should be code transpiled to use ES modules rather than CJS modules, not the untranspiled source.